m3-libs/libm3/src/arith/POSIX/MathPosixC.c

```diff
@@ -0,0 +1,63 @@
+/*
+  Glue to fix broken prototypes in CM3-d5.11.9
+
+  Author : Mika Nystrom <mika.nystroem@intel.com>
+  June, 2024
+
+  Copyright (c) Intel Corporation, 2024.  All rights reserved.
+
+  See COPYRIGHT-INTEL for terms.  
+
+  SPDX-License-Identifier: BSD-3-Clause
+
+ */
+
+#include <math.h>
+#include <complex.h>
+
+double
+MathPosixC__frexp_result_glue(double x)
+{
+  int dummy;
+  
+  return frexp(x, &dummy);
+}
+
+int
+MathPosixC__frexp_exp_glue(double x)
+{
+  int exp;
+
+  (void)frexp(x, &exp);
+
+  return exp;
+}
+
+double
+MathPosixC__cabs_glue(double x, double y)
+{
+  double complex z = x + I * y;
```

Comment:
C99?

Reply:
yeah you know what... it would probably be better to do this differently.  What I was doing here was to make Math.cabs call C's cabs.  But that's really not necessary.  We can just use Math.hypot instead.